### PR TITLE
PR for cart page

### PR DESCRIPTION
**Pull request by narharikale**
I have added the following features
- horizontal product listing
- price details card

ecom-pr-cartPage

### CODE REVIEW

File: pages/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                <span class="material-icons"> favorite_border </span>
+                <div class="badge-with-icon">3</div>
+            </div>
+    
+            <div class="badge-parent">
+                <span class="material-icons"> notifications_none </span>
+                <div class="badge-with-icon">1</div>
+            </div>
+        </div>
+    </nav> -->
```
Comment (shraddha15-creator) [inline pages/cart/cart.html:43]: Remove commented code

**Review by shraddha15-creator — APPROVED** [review]
Clean UI.

File: css/common.css [hunk 2]
```diff
@@ -13,8 +13,23 @@
...
+.card.p-2{
+    padding: 2rem;
+}
+
+.br-1px{
+    border: 1px solid;
+}
+
+.cart-count-num{
+    background: #efefef;
```
Comment (JowelTisso) [inline css/common.css:32]: Prefer to use variables

**Review by JowelTisso — APPROVED** [review]

File: css/common.css (hunk 2 above)
Comment (narharikale) [reply css/common.css:32]: thanks for pointing out 👍
i will fix that

File: pages/cart/cart.html (hunk 1 above)
Comment (narharikale) [reply pages/cart/cart.html:43]: thanks for pointing out 👍
but I kept it for a reason

**Pull request merged**